### ISSUE TITLE
Open image command doesn't work for projects with figures in the sub-directories

### ISSUE THREAD

**Issue by excetara2** [SublimeText/LaTeXTools#708]
The open command tries to open the image files at the root folder not the sub-folders where the figures are stored. This could ideally use the analysis if possible to check to see if there is a `\graphicspath` in the file being worked on to add it to the path.

Alternatively if that doesn't work then this could possibly try the root folder with the graphics path and if that doesn't find a file then try in the current directory of the view. These are the most likely two places to keep any figures.

**Comment by excetara2** [comment]
Should this work properly and whatever the other issue is causing this as well?

**Comment by ig0774** [comment]
Right now, we don't have any support for `\graphicspath`, but we probably should.

**Comment by r-stein** [comment]
When I was writing the open image command the analysis was not available and hence supporting `\graphicspath` would have been to complicated.
Now there is still a problem with analysis, because of the lack of supporting nested brackets in python regex. Currently `\graphicspath{{dir1/}{dir2/}{dir3/}}` would result in an analysis entry like: `{ "command": "graphicspath", "args": "{dir1/", ... }`, althrough all directories are in the first `args` part. Hence supporting more than one directory is still problematic.

**Comment by excetara2** [comment]
I had actually forgot that `\graphicspath` can support multiple directories so it will only be able to analyze the first directory in `graphicspath` then.

Also so the current implementation only checks the path from the root tex file and not also if the file is in the directory of the current active view?

**Comment by r-stein** [comment]
Yes currently it is only relative to the root file. Adding a single directory would not be to hard, but adding several requires an improvement (which is necessary anyway) in the analysis.

**Comment by excetara2** [comment]
I see. Well normally I just put the current sub-folder under root as the `\graphicspath` and then write `{results/fig1}` and `{figures/fig1}` if I have separate folders but full support of `\graphicspath` would be great.
